native_client/definitions.mk

```diff
@@ -98,7 +98,7 @@ SOX_LDFLAGS := $(RASPBIAN)/usr/lib/aarch64-linux-gnu/libsox.so
 PYVER := $(shell python -c "import platform; maj, min, _ = platform.python_version_tuple(); print(maj+'.'+min);")
 PYTHON_PACKAGES      :=
 PYTHON_PATH          := PYTHONPATH=$(RASPBIAN)/usr/lib/python$(PYVER)/:$(RASPBIAN)/usr/lib/python3/dist-packages/
-PYTHON_SYSCONFIGDATA := _PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_m_linux_aarch64-linux-gnu
+PYTHON_SYSCONFIGDATA := $(shell find /usr/lib/python3.*/ -type f -name "*.py" | grep sysconfigdata_)
```

Comment:
what is that ?! there's no way this is going to work.

Reply:
You removed `_PYTHON_SYSCONFIGDATA_NAME=` so this 100% explains your permissions errors ...  I dont think `find` here is a good idea, it's a recipe for breaking stuff silently.